docs/source/tensors.rst

```diff
@@ -8,21 +8,32 @@ torch.Tensor
 A :class:`torch.Tensor` is a multi-dimensional matrix containing elements of
 a single data type.
 
-Torch defines nine CPU tensor types and nine GPU tensor types:
+Torch defines 10 tensor types with CPU and GPU variants:
 
-========================   ===========================================   ===========================   ================================
+========================== ===========================================   ============================= ================================
 Data type                  dtype                                         CPU tensor                    GPU tensor
-========================   ===========================================   ===========================   ================================
+========================== ===========================================   ============================= ================================
 32-bit floating point      ``torch.float32`` or ``torch.float``          :class:`torch.FloatTensor`    :class:`torch.cuda.FloatTensor`
 64-bit floating point      ``torch.float64`` or ``torch.double``         :class:`torch.DoubleTensor`   :class:`torch.cuda.DoubleTensor`
-16-bit floating point      ``torch.float16`` or ``torch.half``           :class:`torch.HalfTensor`     :class:`torch.cuda.HalfTensor`
+point                      ``torch.complex128`` or ``torch.cdouble``
```

Comment:
something wrong with the formatting

Reply:
the rendered documentation looks fine

Reply:
Rendered file here https://github.com/pytorch/pytorch/blob/9ccff98d6d69119878f0f8c418b96394244ab7c8/docs/source/tensors.rst shows third row "point" "torch.complex128 or "torch.double". I doubt that's intended.

Reply:
Hmm, thanks. Fixing